src/main/java/org/molgenis/vcf/decisiontree/filter/SampleAnnotatorImpl.java

```diff
@@ -0,0 +1,44 @@
+package org.molgenis.vcf.decisiontree.filter;
+
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypeBuilder;
+import htsjdk.variant.variantcontext.GenotypesContext;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import java.util.List;
+import org.molgenis.vcf.decisiontree.filter.model.Decision;
+
+public class SampleAnnotatorImpl implements SampleAnnotator {
```

Comment:
The keys VISD/VISDP/VISDL are not really intuitive to me, how about VIPC_S/VIPL_S/VIPP_S so that it is equal to the variant effect fields with a _S (Sample) postfix?
```
##VIPC=VIP decision tree classification
##VIPL=VIP decision tree labels (ampersand separated)
##VIPP=VIP decision tree path (ampersand separated)
```